src/EventEndingRow.js

```diff
@@ -74,17 +74,20 @@ class EventEndingRow extends React.Component {
   }
 
   renderShowMore(segments, slot) {
-    let { localizer } = this.props
+    let { localizer, textSaysShowAll } = this.props
```

Comment:
For this, I was thinking that the user could supply their own text. Since our designs might change (e.g. it currently says "View more"), we can make it so that the user can change the show more text instead of always having to update it here. We probably want to make it a function since we can provide `count` to that function.

Reply:
it is already a function (`src/utils/messages.js` line 19-20), but where do you want the user to supply text for the message?

Reply:
I would say being able to provide a third argument to `renderShowMore` where you can render your own text would be good here.

An example case would be when we want to dynamically display text depending on locale. I know there's some locale built into this package, but I haven't really looked into how it works.